pkg/providers/ovirt/mapper/mapper.go

```diff
@@ -594,11 +594,10 @@ func (o *OvirtMapper) mapPlacementPolicy() *kubevirtv1.EvictionStrategy {
 }
 
 func (o *OvirtMapper) mapTimeZone() *kubevirtv1.Clock {
-	clock := &kubevirtv1.Clock{}
-	if timezone, ok := o.vm.TimeZone(); ok {
-		timezoneName, _ := timezone.Name()
-		clockOffsetTimezone := kubevirtv1.ClockOffsetTimezone(timezoneName)
-		clock.Timezone = &clockOffsetTimezone
+	clock := kubevirtv1.Clock{
+		ClockOffset: kubevirtv1.ClockOffset{
+			UTC: &kubevirtv1.ClockOffsetUTC{},
```

Comment:
Why not to use `o.vm.UtcOffset()` here?

Reply:
Good point. I think it can be used if present.

Reply:
Actually - I don't see a way to make this field non-null from oVirt UI (where our cirrosvm lives). Do you know how to do it (besides API)?

Reply:
No, I only checked the API whether there is something we can use.